xml/System.Buffers/ReadOnlySequence`1.xml

```diff
@@ -148,12 +148,22 @@ The consumer is expected to manage the lifetime of memory until <see cref="T:Sys
         <Parameter Name="endIndex" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="startSegment">To be added.</param>
-        <param name="startIndex">To be added.</param>
-        <param name="endSegment">To be added.</param>
-        <param name="endIndex">To be added.</param>
+        <param name="startSegment">The initial node of the linked memory list.</param>
+        <param name="startIndex">The position to the start of the sequence inside <paramref name="startSegment" />.</param>
+        <param name="endSegment">The final node of the linked memory list.</param>
+        <param name="endIndex">The position to the end of the sequence inside <paramref name="endSegment" />.</param>
         <summary>Creates an instance of a <see cref="T:System.Buffers.ReadOnlySequence`1" /> from a linked memory list represented by start and end segments and the corresponding indexes in them.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException"><paramref name="startSegment" /> or <paramref name="endSegment" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException">The running index of <paramref name="startSegment" /> is greater than the running index of <paramref name="endSegment" />, even though <paramref name="startSegment" /> is different to <paramref name="endSegment" />.
+
+-or-
+
+<paramref name="startSegment" /> is equal to <paramref name="endSegment" /> but <paramref name="endIndex" /> is smaller than <paramref name="startIndex" />.
+
+-or-
+
+<paramref name="startIndex" /> is greater than the length of the underlying memory block of <paramref name="startSegment" />.</exception>
```

Comment:
unrelated to your changes: line 191 is missing a period

Reply:
Nice catch, thanks. Fixed.